deployability/modules/testing/testing.py

```diff
@@ -77,7 +77,7 @@ def _run_tests(cls, test_list: list[str], ansible: Ansible, extra_vars: ExtraVar
         """
         for test in test_list:
             rendering_var = {**extra_vars, 'test': test}
-            template = str(ansible.playbooks_path / cls._test_template)
+            template = str(ansible.testing_playbook_path / cls._test_template)
```

Comment:
Now that the playbooks are inside the _test module_ I think the playbooks path should be defined in the module itself. The same applies for the provision

Reply:
Done https://github.com/wazuh/wazuh-qa/pull/4925/commits/af65f6180452ad248b2af0dcf996caec9cd9d768